.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+# JIRA-123 One Sentence Overview
+
+**Work in progress. Don't merge!** State it here and use labels.
+
+## Links
+
+* http://wikia-inc.atlassian.net/browse/JIRA-123
```

Comment:
If we are using markdown we can use this pattern for links
`[Ticket](https://wikia-inc.atlassian.net/browse/<ticket>)`